manifests/l/LAMMPS/LAMMPS/2022.5.4.1/LAMMPS.LAMMPS.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: LAMMPS.LAMMPS
+PackageVersion: 2022.5.4.1

```

```suggestion
PackageVersion: 4May2022
```
This must match Control Panel.

---

You will also need to change the `2022.5.4.1` folder to be `4May2022`